src/misc/messages.cpp

```diff
@@ -88,12 +89,11 @@ class Message {
 			rendered_msg = convert_ansi_markup(markup_msg.c_str());
 
 		assert(rendered_msg.length());
-		const uint16_t cp = UTF8_GetCodePage();
+		const uint16_t cp = get_utf8_code_page();
 		if (rendered_msg_by_codepage[cp].empty()) {
-			if (!UTF8_RenderForDos(rendered_msg,
-			                       rendered_msg_by_codepage[cp],
-			                       cp))
+			if (!utf8_to_dos(rendered_msg, rendered_msg_by_codepage[cp], cp)) {
 				LOG_WARNING("LANG: Problem rendering string");
```

Comment:
(ditto)

`LANG: Problem rendering string from '%s' to '%s'`

Reply:
Changed, but I'm tracing only the source string. Destination string might contain something the host OS (if it is UTF-8 aware) does not know how to handle.